frontend/src/components/keynoters-section/keynote-slide.tsx

```diff
@@ -0,0 +1,108 @@
+/** @jsxRuntime classic */
+
+/** @jsx jsx */
+import { Box, Flex, Heading, jsx, Text } from "theme-ui";
+
+import Image from "next/image";
+
+import { Link } from "~/components/link";
+import { KeynoteSpeaker } from "~/types";
+
+type KeynoteSlide = {
+  title: string;
+  slug: string | null;
+  speakers: KeynoteSpeaker[];
+  standalone?: boolean;
+};
+
+export const KeynoteSlide = ({
+  title,
+  slug,
+  speakers,
+  standalone = false,
+}: KeynoteSlide) => {
+  const image = speakers[0].photo;
+  const highlightColor = speakers[0].highlightColor;
+  const Wrapper = slug ? Link : Box;
+  return (
+    <Wrapper
+      path="/keynotes/[slug]"
+      params={{
+        slug,
+      }}
+      sx={
+        !standalone
+          ? {
+              "&:last-child": {
+                borderRight: "primary",
+              },
+            }
+          : undefined
+      }
+    >
+      <Box
+        sx={{
+          position: "relative",
+          borderLeft: "primary",
+          paddingBottom: "100%",
+          ...(standalone
+            ? {
+                borderRight: "primary",
+                borderTop: "primary",
+                borderBottom: "primary",
+              }
+            : {
+                height: "100%",
```

Comment:
ugly fix, something is wrong with the slider, but it isn't worth fixing since we should move to the styleguide anyway

the only reason I didn't build this new UI with the styleguide is that I didn't have enough energy to work on a separate repo, but we should start building any new UI with it!

In the next few days ago I will have a look